File: module_project/MultipleViewApplication/Objective-C/MultipleViewApplication/Pages/Home/Controller/ViewController.m
//
//  ViewController.m
//  MultipleViewApplication
//
//  Created by 小口 哲史 on 2017/01/24.
//  Copyright © 2017年 小口 哲史. All rights reserved.
//
#import "ViewController.h"
@interface ViewController ()
@end
@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    gvm = [[GridViewManager alloc] init];
    gvm.mainCon = self;
    gvm.delegate = self;
    tableview.delegate = gvm;
    tableview.dataSource = gvm;
    tableview.scrollEnabled = false;
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

-(void)handleA {
    [[TransitionManager sharedEntry] transitionPush:self.navigationController identifier:CONTROLLER_A];
}

-(void)handleB {
    [[TransitionManager sharedEntry] transitionPush:self.navigationController identifier:CONTROLLER_B];
}


@end

File: module_project/InputApplication/Swift/InputViewApplication/Pasge/inputView/Controller/ViewController.swift
//
//  ViewController.swift
//  InputViewApplication
//
//  Created by 小口 哲史 on 2017/12/14.
//  Copyright © 2017年 小口 哲史. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    var iVMgr:InputViewManager = InputViewManager()
    @IBOutlet weak var tv: UITableView!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        iVMgr.setNib(tb:tv, nibName:"InputViewCell")
        tv.delegate = iVMgr
        tv.dataSource = iVMgr
        iVMgr.tv = tv
        iVMgr.tv?.isScrollEnabled = false
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
}

